contracts/LoanFactory.sol

```diff
@@ -1,386 +1,83 @@
 pragma solidity 0.4.24;
 
-/**
- * @title Ownable
- * @dev The Ownable contract has an owner address, and provides basic authorization control
- * functions, this simplifies the implementation of "user permissions".
- */
-contract Ownable {
+import "./Loan.sol";
+
+contract LoanFactory {
     address public owner;
+    address public worker;
+    address[] public loans;
+    uint256 public loanId;
+    mapping(bytes32 => uint256) public marginLeadTime;
+    mapping(bytes32 => uint256) public matureLeadTime;
+    mapping(bytes32 => uint256) public interestLeadTime;
 
-    event OwnershipTransferred(address indexed previousOwner, address indexed newOwner);
+    event NewLoan(address indexed loan, uint256 loanId);
```

Comment:
recommended way to name events is to prepend them with `Log` i.e. `LogNewLoan` instead of `NewLoan`
ref: https://github.com/ConsenSys/smart-contract-best-practices/blob/12563e66368bad1ae97f7798b28751b2d1820a2c/docs/recommendations.md#differentiate-functions-and-events

Reply:
I think the addition of `emit` for events is sufficient to differentiate the names, previously it was less distinct